src/03-control-flow.js

```diff
@@ -86,7 +114,15 @@ function positiveNegativeOrZero(a) {}
  *  aroundTheWorldGreeting();
  *  //> "Hello World"
  */
-function aroundTheWorldGreeting(language) {}
+function aroundTheWorldGreeting(language) {
```

Comment:
We can give our function a default value here: `function aroundTheWorldGreeting(language="english")`